Example/Optik/ViewController.swift

```diff
@@ -34,9 +62,24 @@ internal final class ViewController: UIViewController {
         let urls = [url1, url2, url3, url4]
         let imageDownloader = AlamofireImageDownloader()
         
-        let viewController = Optik.imageViewerWithURLs(urls, imageDownloader: imageDownloader)
+        let viewController = Optik.imageViewer(withURLs: urls, imageDownloader: imageDownloader)
```

Comment:
This custom transition isn't supported for remote images just yet. I will update this in a separate PR.